### DISCUSSION TITLE
[Question]

### DISCUSSION THREAD

**Discussion by Bronya0** [flet-dev/flet#3160] [Q&A]
### Question

The app built by flet pack under Windows cannot be pinned to the taskbar and started normally. In fact, it is flet.exe that is pinned to the taskbar, not my real program window, which makes it impossible to start from the taskbar later，

### ------------------------------------------------------

- [ ] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
`flet pack` creates a wrapper around original Flet executable.

Use `flet build windows` command instead: https://flet.dev/docs/publish
